src/blockchain/ain/jellyfish/utils/raw-tx-utxo.ts

```diff
@@ -0,0 +1,96 @@
+import { Vout, Script } from '@defichain/jellyfish-transaction';
+import BigNumber from 'bignumber.js';
+import { Config } from 'src/config/config';
+import { UtxoConfig } from '../domain/entities/utxo-config';
+import { UtxoSizePriority } from '../domain/enums';
+import { RawTxDto } from '../dto/raw-tx.dto';
+import { RawTxBase } from './raw-tx-base';
+import { RawTxUtil } from './raw-tx-util';
+
+export class RawTxUtxo extends RawTxBase {
+  // SEND UTXOS //
+  async sendFromTo(from: string, to: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, to, amount, true, { useFeeBuffer: false, sizePriority: UtxoSizePriority.SMALL }),
+    );
+  }
+
+  async sendFromLiq(to: string, amount: BigNumber, sizePriority: UtxoSizePriority): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(Config.staking.liquidity.address, to, amount, true, { useFeeBuffer: true, sizePriority }),
+    );
+  }
+
+  async sendToLiq(from: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, Config.staking.liquidity.address, amount, false, {
+        useFeeBuffer: false,
+        sizePriority: UtxoSizePriority.SMALL,
+      }),
+    );
+  }
```

Comment:
I would refactor this a little bit. The logic in here does not really depend on if it's to or from liquidity address but on the type of transaction. I think there should always be a from and to parameter (to keep it reusable for the future).
- `sendFromLiq` is something like a sendMany
- `sendToLiq` is something like a sendUtxo
- `sendFromTo` is a sendmany without fee buffer, because amounts are very small ... so maybe name it sendSmallUtxo or sendFeeUtxo or similar?

Reply:
IMHO I would like to have collective functionalities and I would NOT structure them like we jellyfish or CLI has structured them. Why? Because we have other needs. In our cases we are sending from OR to LIQ.

Again I don't think that the current solution is perfect and the naming can be improved. But moving the logic out of this class is also something I would not do. If we are only exposing a `send` function in here, we have duplications all over the place.